server/controllers/UsersController.js

```diff
@@ -101,6 +103,128 @@ class UsersController {
         }
       }));
   }
+
+  /**
+ * @description - Route handler to send password reset link
+ * @param {object} req - request object
+ * @param {object} res - response object
+ * @returns {object} - returns object
+ */
+  static forgotPassword(req, res) {
+    const { email } = req.body;
+    const lifeSpan = '1h'; // token expires after 1 hour
+
+    User.findOne({
+      where: {
+        email
+      }
+    })
+      .then((userFound) => {
+        if (!userFound) {
+          return res.status(404).json({
+            errors: {
+              message: ['No account with that email address exists']
+            }
+          });
+        }
+
+        const userId = userFound.id;
+        const token = createToken(userId, lifeSpan);
+        // eslint-disable-next-line max-len
+        const forgotUrl = `${req.protocol}://${req.get('host')}/api/v1/users/reset/${token}`;
```

Comment:
we are now using an env variable for the apps base url right? I think this should be updated too if we're going with that